Request/ParamFetcher.php

```diff
@@ -95,6 +96,8 @@ public function getParams()
 
     /**
      * {@inheritdoc}
+     *
+     * @throws \InvalidArgumentException
```

Comment:
I don't think this is needed here. the case where it happens is not a case where you would want to catch it (except on a top-level error handler)